js/global.js

```diff
@@ -1,8 +1,8 @@
 const canvas = document.getElementById("c");
 const ctx = canvas.getContext("2d");
 
-deviceWidth = window.innerWidth;
-deviceHeight = window.innerHeight;
+let deviceWidth = window.innerWidth;
```

Comment:
I think that `deviceWidth` and `deviceHeight` should be const.

Reply:
They could both change if you resize the window.